github-workflow/aks-deploy.yaml

```diff
@@ -68,6 +68,7 @@ jobs:
         creds: ${{ secrets.AZURE_CREDENTIALS }}
 
     # Deploy the cluster into your environment, assuming all prerequisites are up and running.
+    # It is possible to restrict the ips, from where the kubernetes API server is accessed, by setting the clusterAuthorizedIPRanges parameter. By default, full access by all ips.
```

Comment:
We do not use to have these so specific comments in here. Please let’s consider removing it from here but feel free to add as many comment as required from the deploy instructions option 3